exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -75,8 +73,8 @@ print(2 * 2) // Prints 4
 ### Functions
 
 In Swift, [functions][functions] are a chunk of code that performs a task.
-A function has a return type and can thereby be used as a value and be parsed as arguments to other functions.
-In Swift are functions defined using the `func` keyword followed by the name of the function, arguments separated by commas, and a return type.
+A function has a return type and can thereby be used as a value and be passed as an argument to other functions.
+In Swift functions are defined using the `func` keyword followed by the name of the function, arguments separated by commas, and a return type.
```

Comment:
This sounds a bit weird. I think a comma would improve readability a bit e.g:

In Swift, functions are defined using the `func` keyword followed by the name of the function, arguments separated by commas, and a return type.